packages/contracts-bedrock/scripts/getting-started/config.sh

```diff
@@ -81,7 +81,8 @@ config=$(cat << EOL
   "eip1559DenominatorCanyon": 250,
   "eip1559Elasticity": 6,
 
-  "l2GenesisDeltaTimeOffset": null,
+  "l2GenesisEcotoneTimeOffset": "0x0",
+  "l2GenesisDeltaTimeOffset": "0x0",
```

Comment:
Because `l2GenesisEcotoneTimeOffset` was missing, the `genesis.json` file generated by the `op-node` genesis CLI was missing `config.ecotoneTime`.

Reply:
Note that if we activate Ecotone at genesis, we also want to set different values for `scalar` and `overhead`. The `overhead` should be `0` and the scalar could e.g. be the current OP mainnet encoded scalar value `0x010000000000000000000000000000000000000000000000000c5fc500000558`. See https://github.com/ethereum-optimism/superchain-ops/pull/92 for details of how this value was generated using `op-chain-ops/cmd/ecotone-scalar`.

There's ongoing work to simplify the deploy config to have specific fields for the `baseFeeScalar` and `blobBaseFeeScalar` as to not be required to encode them manually into the `scalar` field.

Reply:
https://github.com/ethereum-optimism/optimism/pull/10143 fixes the ambiguity with scalar here